src/main.js

```diff
@@ -2170,25 +2162,6 @@ function BlackboxLogViewer() {
             }
         });
 
-        // New workspaces feature; local storage of user configurations
-        prefs.get('workspaceGraphConfigs', function(item) {
-            if (item) {
-                workspaceGraphConfigs = upgradeWorkspaceFormat(item);
-            } else {
-                workspaceGraphConfigs = defaultWorkspaceGraphConfigs;
-            }
-        });
-
-        prefs.get('activeWorkspace', function (id){
-            if (id) {
-                activeWorkspace = id
-            }
-            else {
-                activeWorkspace = 1
-            }
-        });
-        onSwitchWorkspace(workspaceGraphConfigs, activeWorkspace);
-
```

Comment:
This is handled somewhere else or no longer needed?

Reply:
src/main.js:1072

Reply:
Yes this code exists above

Reply:
Yes was lazy to search for myself :see_no_evil:

Reply:
Yes this code exists above